service/middleware.go

```diff
@@ -80,14 +80,16 @@ func MustHaveUser(r *http.Request) *user.DBUser {
 
 // ToPluginContext creates a UIContext from the projectContext data.
 func (pc projectContext) ToPluginContext(settings evergreen.Settings, dbUser *user.DBUser) plugin.UIContext {
+	project, err := pc.GetProject()
+	grip.Error(errors.Wrap(err, "unable to fetch project"))
 	return plugin.UIContext{
```

Comment:
I'd make plugin.UIContext compose projectContext, so that the method/cache is exposed.

Reply:
projectContext has additional fields which we do not use.